queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name like '%mon';
+SELECT name FROM animals WHERE date_of_birth > '2016-01-01' AND date_of_birth < '2019-01-01';
+SELECT name FROM animals WHERE neutered = true AND escape_attempts > 3;
```

Comment:
- [ ] Please here is `less than` 3 escape attempts: `escape_attempts < 3` (List the name of all animals that are neutered and have less than 3 escape attempts)

Reply:
That was a mistake, great job noticing that.